docusaurus/docs/user-docs/getting-started/user-guide-fundamentals.md

```diff
@@ -0,0 +1,19 @@
+---
+title: User Guide fundamentals
+displayed_sidebar: userDocsSidebar
+description: Here are some concepts to keep in mind
+---
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development, Staging or Production Environment** <br/> When you start working on your application, it is in a development environment, which is the status for the content structure and application configuration. After deploying your application, it is in production or staging environment. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-type Builder. In this user guide the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
+
+- **Versions** <br/> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version. However, we always keep live both the documentation of the current Strapi version, and the documentation of the previous one - the latter being officially and actively maintained for up to 12 months after the release of the newest Strapi version.
+
+- **License and Pricing Plans** <br/> As a Strapi user you have the choice between using the Community Edition, which is entirely free, or the [Enterprise Edition](https://strapi.io/pricing-self-hosted). In this user guide, if a feature is only available for the Enterprise Edition, an <EnterpriseBadge /> badge is displayed beside the section's title.
```

Comment:
I noticed that the introduction/Strapi fundamentals page still doesn't mention the new pricing Cloud badges, but if I understand correctly: both PRs should be released at the same time, there's no explanation of the badges in the badges PR either, and it's in the User Guide that those Cloud badges are mostly used.
I think that we should definitely add something in here to make sure users understand. @pwizla what do you think?

Reply:
Yes, Mégane, I definitively agree about adding information for Cloud badges too (see [my similar comment](https://github.com/strapi/documentation/pull/2024#discussion_r1505605505)).